bot/utils/christmas/christmasifications.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from random import choice, randint
+
+from PIL import Image
+
+log = logging.getLogger()
+
+
+def hat(im: Image) -> Image:
+    """Adds a Christmas hat to the image."""
+    im = im.convert("RGB")
+    wt, ht = im.size
+    hat = Image.open("bot/resources/holidays/christmas/christmas-hat.png")
+    hat_size = randint(wt // 10, wt // 7)
+    rot = randint(0, 90)
+    hat = hat.resize((hat_size, hat_size))
+    hat = hat.rotate(rot)
+    x = randint(wt - (hat_size * 3), wt - hat_size)
+    y = randint(10, hat_size)
+    im.paste(hat, (x, y), hat)
+    im.paste(hat, (x + hat_size, y + (hat_size // 4)), hat)
+    im.paste(hat, (x - hat_size, y - (hat_size // 2)), hat)
+    return im
+
+
+def snowflake(im: Image) -> Image:
+    """
+    Adds a snowflake to the image.
+
+    The snowflake is of a size at least one-fifths that of the original image and may be rotated
+    up to 90 degrees anti-clockwise.
+    """
+    im = im.convert("RGBA")
+    wt, ht = im.size
+    snowflake = Image.open("bot/resources/holidays/christmas/snowflake.png").convert("RGBA")
+    snowflake_size = randint(wt // 10, wt // 7)
+    rot = randint(0, 90)
```

Comment:
might not need a new variable as rot, we can use as 

```py
snowflake = snowflake.rotate(randint(0,90))
```

Reply:
Good suggestion, I agree. Will fix.